src/classes/LineItemTriggerHandler.cls

```diff
@@ -16,6 +16,7 @@ public with sharing class LineItemTriggerHandler {
 		
 		// load required data: 1) related orders 2) force reload all line items for those orders
 		List<Order__c> orders = [SELECT Purchased__c,Product_Category_1__c,Product_Category_Units_1__c,Product_Category_2__c,Product_Category_Units_2__c FROM Order__c WHERE Id in :orderIds];
+		List<Order__c> myVariable = [SELECT Purchased__c,Product_Category_1__c,Product_Category_Units_1__c,Product_Category_2__c,Product_Category_Units_2__c FROM Order__c];
```

Comment:
The result of this query is not bound. Please add either a `LIMIT` or a `WHERE` statement to the query to limit the number of results.